File: Python/Ex17.py
a= float(input("Valor A: "))
b= float(input("Valor B: "))
c= float(input("Valor C: "))

if (a * a == b * b + c * c) or (b * b == c * c + a * a) or (c * c == a * a + b * b):
 print("Triangulo Retangulo")
else :
    print("Nao eh um Triangulo Retangulo") 

File: C/Ex17.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    float a, b, c;

    printf("Valor A: ");
    scanf("%f", &a);
    printf("Valor B: ");
    scanf("%f", &b);
    printf("Valor C: ");
    scanf("%f", &c);

     if ((a * a == b * b + c * c) ||
        (b * b == c * c + a * a) ||    
        (c * c == a * a + b * b)) printf("Triangulo Retangulo\n");
         
    
           else{
           printf("Nao eh um Triangulo Retangulo");
       }
return 0;

}
        